packages/venia-ui/package.json

```diff
@@ -34,6 +34,7 @@
   "homepage": "https://github.com/magento/pwa-studio/tree/master/packages/venia-ui#readme",
   "dependencies": {
     "@babel/template": "~7.8.6",
+    "@formatjs/intl-numberformat": "~5.6.3",
```

Comment:
Is this dependency still required?

Reply:
Missed this, fixed.
I've changed `grid-template-columns: auto auto` to `grid-auto-flow: column` as @jimbo suggested, but left `justify-content: space-between;` so columns aligned to the far ends per design.
Thank you